.changeset/olive-elephants-protect.md

```diff
@@ -0,0 +1,17 @@
+---
+"@comet/admin": major
+---
+
+Remove `DialogContent` from `EditDialog` as spacing inside a dialog is not commonly needed in the Comet DXP design
+
+To maintain the current styling of `EditDialog`, add `DialogContent` as a child.
```

Comment:
Maybe a more detailed example or explanation could be helpful, both in the changeset and the migration-guide. 

E.g., that forms/text do need the spacing and grids do not. Or that elements that create their own spacing (such as DataGrid) shouldn't have additional spacing.